pyproject.toml

```diff
@@ -200,3 +200,7 @@ testpaths = "tests"
 # use `pytest-mock` drop-in replacement for `unittest.mock`
 # https://pytest-mock.readthedocs.io/en/latest/configuration.html#use-standalone-mock-package
 mock_use_standalone_module = false
+# https://docs.pytest.org/en/7.1.x/how-to/logging.html#how-to-manage-logging
+# uncomment to adjust captured log levels
+# WARNING some tests cli v012 tests may fail if this is altered
+# log_level = "info"
```

Comment:
This was a bit annoying these tests fail if this `log_level` is set at all. Even if it's the pytest default of `warning`
https://docs.pytest.org/en/7.1.x/how-to/logging.html#how-to-manage-logging

Looking forward to @Shinnnyshinshin 's https://github.com/great-expectations/great_expectations/pull/6355

```
FAILED tests/cli/v012/test_datasource_pandas.py::test_cli_datasource_profile_with_datasource_arg
FAILED tests/cli/v012/test_datasource_pandas.py::test_cli_datasource_profile_with_no_datasource_args
FAILED tests/cli/v012/test_datasource_pandas.py::test_cli_datasource_profile_with_skip_prompt_flag
FAILED tests/cli/v012/test_datasource_pandas.py::test_cli_datasource_profile_with_additional_batch_kwargs
FAILED tests/cli/v012/test_datasource_pandas.py::test_cli_datasource_profile_with_valid_data_asset_arg
FAILED tests/cli/v012/test_datasource_sqlite.py::test_cli_datasource_profile_with_datasource_arg[test_backends0]
FAILED tests/cli/v012/test_datasource_sqlite.py::test_cli_datasource_profile_with_datasource_arg_and_generator_name_arg[test_backends0]
FAILED tests/cli/v012/test_datasource_sqlite.py::test_cli_datasource_profile_with_no_datasource_args[test_backends0]
FAILED tests/cli/v012/test_datasource_sqlite.py::test_cli_datasource_profile_with_data_asset_and_additional_batch_kwargs_with_limit[test_backends0]
FAILED tests/cli/v012/test_datasource_sqlite.py::test_cli_datasource_profile_with_valid_data_asset_arg[test_backends0]
= 10 failed, 404 passed, 24 skipped, 18 xfailed, 45 warnings in 352.23s (0:05:52) 
```